### PR TITLE
fix(<Updateable />): remove shouldComponentUpdate

### PR DESCRIPTION
**Pull request by robineng**
remove shouldComponentUpdate of Updateable component as it does not update on ticksize change. It is
also hard to have a shouldComponentUpdate in this component as it's not bound to a specific set of
props.

### CODE REVIEW

**Review by iamstarkov — APPROVED** [review]

File: src/components/updateable/updateable.jsx [hunk 1]
```diff
@@ -27,17 +29,13 @@ export default class Updateable extends React.Component {
...
-  }
-
   updateClass(oldValue, newValue) {
     const diff = newValue - oldValue;
     if (diff === 0) return;
 
     this.setState(
       {
-        diff, // eslint-disable-line
+        diff,
```
Comment (bstream) [inline src/components/updateable/updateable.jsx:38]: Maybe we can remove `diff` from the internal state? it seems like it is not used anyway

Comment (robineng) [reply src/components/updateable/updateable.jsx:38]: I think it's optionally used in the component that this component renders with it's render prop but maybe @jesperwiner knows?

Comment (robineng) [reply src/components/updateable/updateable.jsx:38]: Yes it's used outside this component. I think that this component would be much easier to understand if it was an HOC

Comment (bstream) [reply src/components/updateable/updateable.jsx:38]: Oh yeah I see now in the comment below `/** (props, {diff, updateableClass, value}=state) => <Component /> */`. Hmm ok lets go with this then :)

**Review by bstream — APPROVED** [review]
lgtm

**Pull request merged**